provider/datagen/data/grapheme_cptrie.toml

```diff
@@ -0,0 +1,424 @@
+# This file is part of ICU4X. For terms of use, please see the file
```

Comment:
Suggestion: put these in `data/cache` or `data/precomputed` or something to make clear that this is not source data.